rebasehelper/tests/test_utils.py

```diff
@@ -150,6 +150,30 @@ def write():
         assert capturer.stdout == 'test stdout'
         assert capturer.stderr == 'test stderr'
 
+    @pytest.mark.parametrize('specification, expected_result', [
+        ('rgb:0000/0000/0000', (0, 0, 0, 4)),
+        ('rgb:ffff/ffff/ffff', (65535, 65535, 65535, 4)),
```

Comment:
I think using hexadecimal constants would be more appropriate. Same for the other test.